doc/source/catalog_generation.rst

```diff
@@ -286,6 +288,15 @@ these values. Specific flag values are defined below in table 2:
     |            | or other region with a low number of input images         |
     +------------+-----------------------------------------------------------+
 
+.. attention::
+
+    The final output filter-specific sourcelists do not contain all detected sources. Sources that are considered
+    scientifically dubious are filtered out and not written to the final source catalogs. For all detectors other than
+    WFC3/IR, sources with a flag value greater than 5 are filtered out. For WFC3/IR, sources with a flag value of
+    65368 are filtered out. Users can adjust this value using a custom input parameter file and changing the
```

Comment:
Maybe the flag value got changed in the code, but it also needs to be changed to "5" here.